subcommand/delete-completed-job/command_test.go

```diff
@@ -34,7 +34,7 @@ func TestRun_ArgValidation(t *testing.T) {
 		},
 		{
 			[]string{"-k8s-namespace=default", "-timeout=10jd", "job-name"},
-			"\"10jd\" is not a valid timeout: time: unknown unit jd in duration 10jd",
+			"\"10jd\" is not a valid timeout",
```

Comment:
I'm running go 1.15 now and these errors are a bit different. If we remove this part then they should work with 1.14 and 1.15